parser/unparser_formatting_options.go

```diff
@@ -0,0 +1,52 @@
+package parser
+
+import (
+	"fmt"
+
+	"github.com/google/cel-go/common/operators"
+)
+
+// Defined defaults for the formatting options
+var (
+	DefaultWrapColumn = 80
```

Comment:
If you need to export this, make sure to document.

Reply:
Unexported the field